### ISSUE TITLE
Pass command line arguments

### ISSUE THREAD

**Issue by timster** [nklayman/vue-cli-plugin-electron-builder#185]
Is it possible to pass command line arguments to the electron process?

I was hoping to be able to do:

```
yarn electron:serve --param1 --param2=value
```

I need to have an app that accepts the arguments when it's built as well (for this you would need to use `--` for param passthrough).

```
./dist_electron/mac/testapp.app/Contents/MacOS/testapp -- --param1 --param2=value
```

I would be glad to help implement this if it doesn't already exist.

**Comment by nklayman** [comment]
You can [set main process args for development](https://nklayman.github.io/vue-cli-plugin-electron-builder/guide/configuration.html#webpack-configuration) (in the example), but not for a production build.

**Comment by nklayman** [comment]
That would have to be handled by [electron-builder](https://electron.build), as it handles the packaging step.

**Comment by timster** [comment]
I should have been more specific. It does work when running the bundled app.

I add this line to `background.js`:
```javascript
console.log('ARGS', process.argv)
```
The output when running the above command for dev is this:

```
ARGS [ '/testapp/node_modules/electron/dist/Electron.app/Contents/MacOS/Electron', 'dist_electron' ]
```

And running with the bundled app is this:
```
ARGS [ '/testapp/dist_electron/mac/example.app/Contents/MacOS/testapp', '--param1', '--param2=value' ]
```

So the passthrough of arguments in dev would need to be handled by this plugin. Something like this seems to work (where mainProcessArgs is currently defined).

```javascript
const mainProcessArgs = pluginOptions.mainProcessArgs || []
mainProcessArgs.push(...process.argv.slice(3))
```

**Comment by nklayman** [comment]
The issue is that some args are used by the dev server. These could be filtered out though. I will look into implemting this

**Comment by nklayman** [comment]
Dashboard, debug, and headless would all have to be removed.

**Comment by nklayman** [comment]
Fixed with 0e2b395cc261237135099e77110e501519f609c0. Releasing rc.11 now, upgrade and try, let me know if it works.
